rdflib/plugins/serializers/trig.py

```diff
@@ -83,7 +68,10 @@ def serialize(self, stream, base=None, encoding=None,
             if self.default_context and store.identifier==self.default_context:
                 self.write(self.indent() + '\n{')
             else:
-                self.write(self.indent() + '\n<%s> {' % self.getQName(store.identifier))
+                iri = self.getQName(store.identifier)
+                if iri is None:
+                    iri = '<%s>' % store.identifier
+                self.write(self.indent() + '\n%s {' % iri)
```

Comment:
getQName gets to return None if it has no abbreviation, which I test with the URIRef("http://example.com/foo.") case.

As I understand it, a non-None iri (e.g. a successful abbreviation like "ns1:foo") shouldn't be wrapped in <>. That seemed to be a bug before.  See Example 1 in http://www.w3.org/TR/trig/#sec-triple-statements for a demo of no angle brackets.

I pushed an update to get bnode graph ids right too (like "_:a1b2c3 { ... }").